### PR TITLE
Fix #17

### LINKED ISSUE #17: highstock.js error breaks dashboard
Was working yesterday and this morning. Timestamps show no change since last week.

Error is

Error: Invalid value for <rect> attribute x="NaN" highstock.js:51
Error: Invalid value for <rect> attribute width="NaN" highstock.js:51
Error: Invalid value for <rect> attribute x="NaN" highstock.js:51
Error: Invalid value for <rect> attribute width="NaN" highstock.js:51
Uncaught TypeError: Cannot read property 'substr' of undefined highstock.js:18

[mchew@people1.dmz.scl3 public_html]$ ls -al pinning_dashboard/
total 100
drwxr-xr-- 2 mchew apache  4096 May 21 11:14 .
drwxr-s--- 5 mchew apache  4096 Jul  1 17:01 ..
-rw-r--r-- 1 mchew apache  6392 Jun 26 17:16 chartoptions.js
-rw-r--r-- 1 mchew apache  7847 Jun 26 17:21 dashboard.js
-rw-r--r-- 1 mchew apache  1007 Jun 18 14:02 index.html

It could be a highstock push or telemetry dirty data.

### PR DESCRIPTION
**Pull request by monicachew**
Fixes error by getting rid of mindate (?!). I don't totally understand it, but it seems to work.

Also filter duplicate dates where Telemetry is reporting multiple counts for the same day.

### CODE REVIEW

**Comment by monicachew** [comment]
@mozkeeler this is pushed

File: dashboard.js [hunk 1]
```diff
@@ -90,6 +91,31 @@ function sortByDate(p1, p2)
   return p1[0] - p2[0];
 }
 
+// Filter duplicate dates to account for screwed up telemetry data
+function filterDuplicateDates(series)
+{
+  // Work on a copy
+  var s = series;
```
Comment (mozkeeler) [inline dashboard.js:98]: Isn't s just an alias of series? I think s = series.slice() would work.

**Comment by mozkeeler** [comment]
Sorry I took so long to get to this - for some reason I misunderstood what you meant. Anyway, looks good except for that one comment.

**Comment by monicachew** [comment]
You are right, slice already returns a copy.

**Comment by monicachew** [comment]
Well, except that I wanted to work on a copy to avoid side-effects. Also added event tags for dropbox.

**Pull request merged**